packages/components/src/slot-fill/fill.js

```diff
@@ -26,25 +31,21 @@ class Fill extends Component {
 		if ( ! this.occurrence ) {
 			this.occurrence = ++occurrences;
 		}
-		const { getSlot = noop } = this.context;
+		const { getSlot } = this.props;
 		const slot = getSlot( this.props.name );
 		if ( slot && ! slot.props.bubblesVirtually ) {
 			slot.forceUpdate();
 		}
 	}
 
 	componentWillUnmount() {
-		const { unregisterFill = noop } = this.context;
+		const { unregisterFill = noop } = this.props;
```

Comment:
You can remove `noop` also here for consistency.

Reply:
cb19a4e